src/api/layers.api.js

```diff
@@ -8,31 +8,34 @@ import axios from "axios";
 export const LayerTypes = {
     WMTS: 'wmts',
     WMS: 'wms',
-    GEOJSON: 'geojson'
+    GEOJSON: 'geojson',
+    AGGREGATE: 'aggregate',
 };
 
 /**
  * @class
- * @name layers:TimeSeriesConfig
+ * @name layers:TimeConfig
  *
  * Time configuration for a {@link WMTSLayer} or {@link WMSLayer}. It will determine which "timestamp" to add
  * to the URL used to request tiles/image.
  */
-export class TimeSeriesConfig {
+export class TimeConfig {
 
     /**
      * @param {String} behaviour how the default time series is chosen
      * @param {Array<String>} series list of series identifier (that can be placed in the WMTS URL)
      */
-    constructor(behaviour = "last", series = []) {
+    constructor(behaviour = null, series = []) {
         this.behaviour = behaviour;
-        this.series = series;
-        this.currentTimestamp = null;
-        if (this.series.length > 0) {
-            switch (this.behaviour) {
-                case 'last':
-                    this.currentTimestamp = this.series[0];
-            }
+        this.series = [...series];
+        if (this.behaviour === 'last' && this.series.length > 0) {
+            this.currentTimestamp = this.series[0];
+        } else if (this.behaviour) {
+            this.currentTimestamp = this.behaviour;
+        } else if (this.series.length > 0) {
+            this.currentTimestamp = this.series[0];
+        } else {
+            this.currentTimestamp = 'current';
```

Comment:
maybe add a comment describing the different behaviour that exists (and how the data from backend is looking for these cases)

Reply:
I've pushed some doc, but forgot to switch branch before, so it will come with https://github.com/geoadmin/web-mapviewer/pull/25 ...